Source/Fuse.Elements/Caching/Cache.uno

```diff
@@ -215,31 +216,9 @@ namespace Fuse.Elements
 		{
 			foreach (CacheTile tile in cache.CacheTiles)
 			{
-				draw
-				{
-					float2[] Vertices: new[]
-					{
-						float2(0, 0), float2(0, 1), float2(1, 1),
-						float2(0, 0), float2(1, 1), float2(1, 0)
-					};
-
-					float2 Coord: vertex_attrib(Vertices);
-					float2 TexCoord: float2(Coord.X, 1.0f - Coord.Y);
-
-					float2 Size: float2(tile.Texture.Size.X, tile.Texture.Size.Y) / dc.ViewportPixelsPerPoint;
-
-					float4 p: Vector.Transform(float4(Coord * Size, 0, 1), tile._compositMatrix);
-
-					ClipPosition: Vector.Transform(p, dc.Viewport.ViewProjectionTransform );
-
-					PixelColor: sample(tile.Texture, TexCoord, SamplerState.LinearClamp);
-
-					PixelColor: float4(prev.XYZ*opacity, prev.W*opacity);
-
-					CullFace: dc.CullFace;
```

Comment:
So, this subtly breaks the Viewport.CullFace feature. However, it seems this is pretty heavily broken already; the vast majority of call-sites already ignore it. But I can't really understand why we would need a feature like that in the first place.